.devcontainer/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM mcr.microsoft.com/devcontainers/base:ubuntu
```

Comment:
My first instinct looking at this is that Ubuntu might not be the best base image for this. They have a fairly extensively patched Python (which has led to a lot of papercuts/friction in the [past](https://gist.github.com/tiran/2dec9e03c6f901814f6d1e8dad09528e)) and using a different Linux distro that doesn't have extensive patches (Fedora?) would be nicer.

Or... maybe... having a somehow-unpatched-by-linux-distro Python installed in the environment would be nice. Honestly, I have no idea if the latter is feasible though. Maybe pyenv?

Reply:
How about using `mcr.microsoft.com/devcontainers/python` instead? That should already have a non-distro Python installed.

Available dev containers can be found here:

https://hub.docker.com/_/microsoft-devcontainers